### ISSUE TITLE
Snap or apt package

### ISSUE THREAD

**Issue by arstulke** [allo-/virtual_webcam_background#65]
Is there a way to provide a snap or apt package for easier installation?

**Comment by allo-** [comment]
No. Snap is probably complicated because of its sandboxing (and being a typical canonical project like Mir).

A debian package may be possible. You should be able to get it to run rather easy on Debian by installing the v4l2loopback package (that should prepare the kernel driver), adding your user to the video group, and installing the dependencies in a virtual environment using pip.

Building a debian package should be possible (I think there are good debhelpers for python), but additional configuration is probably needed as debian packages should not modify files like the groups a user is in or a modprobe.d file for loading v4l2loopback automatically, which you would probably want to avoid inserting the kernel module yourself every time. An option would be to ship a configuration script, that allows for easy setup of such configuration files.
